### DISCUSSION TITLE
How to solve this Sphinx warning?

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#13356] [Q&A]
Building Weblate doc in Brazilian Portuguese (pt_BR), I'm getting the following Sphinx warning:

> /home/rffontenelle/Projects/weblate/docs/contributing/index.rst:90: ERROR: Unknown target name: "página de doação". [docutils]

The [source string](https://github.com/WeblateOrg/weblate/blob/9aebd12dbc74cd6ad6f588004a4b671fc4571be7/docs/contributing/index.rst?plain=1#L87-L99) has two occurrences of ``` `donate page`_```, and both of them are correctly translated in Brazilian Portuguese to ``` `página de doação`_``` (search in [pt_BR docs.po](https://raw.githubusercontent.com/WeblateOrg/weblate/refs/heads/main/docs/locales/pt_BR/LC_MESSAGES/docs.po), there is no visible syntax error).

The rendered page shows error for the second occurrence of página de doação ('donate page' translation), as can be seen in [Funding Weblate development](https://docs.weblate.org/pt-br/latest/contributing/index.html#funding-weblate-development):

![image](https://github.com/user-attachments/assets/477e24e5-0f4c-464f-b338-11337fb7b700)

To reproduce:
```shell
# install docs requirements from pyproject.toml
uv sync --no-sources --only-group docs --frozen
. .venv/bin/activate
# download latest translation (other fixes not yet merged)
cd docs
curl -L https://hosted.weblate.org/download/weblate/documentation/pt_BR/ -o locales/pt_BR/LC_MESSAGES/docs.po
make SPHINXOPTS='-Dlanguage=pt_BR' html
```

How can I translate 'donate page' text without breaking Sphinx' hyperlink?

**Comment by nijel** [comment]
It looks like a bug in Sphinx to me, reported at https://github.com/sphinx-doc/sphinx/issues/13186

But when reading the paragraph, I think it deserves rephrasing, so I've changed in also in a way it doesn't hit this issue in 68daaacbbdab135fedaa8ed83669c5c6b603badb.
